doc/src/dump.rst

```diff
@@ -202,7 +202,7 @@ or multiple smaller files).
    to a dump file may be slightly outside the simulation box.
    Re-neighbor timesteps will not typically coincide with the
    timesteps dump snapshots are written.  See the :doc:`dump_modify
-   pbc <dump_modify>` command if you with to force coordinates to be
+   pbc <dump_modify>` command if you wish to force coordinates to be
```

Comment:
This doesn't make sense, I think we should remove "to", so it says "command if you wish force coordinates to be ..."

Reply:
yes, the "to" is needed